### PR TITLE
ChangeSets dupsort

### PR DESCRIPTION
**Pull request by AskAlexSharov**
1 known problem: call_traces stage return "Nonce too high" error on ./cmd/integration,(even if i totally remove cache warmup by changesets...).

### CODE REVIEW

File: common/changeset/account_changeset_utils.go [hunk 1]
```diff
@@ -2,193 +2,35 @@ package changeset
...
+}
 
-	for i := 0; i < n; i++ {
-		_, err = buf.Write(s.Changes[i].Value)
-		if err != nil {
-			return nil, err
+func encodeAccounts2(blockN uint64, s *ChangeSet, f func(k, v []byte) error) error {
+	sort.Sort(s)
+	for _, cs := range s.Changes {
+		newK := dbutils.EncodeBlockNumber(blockN)
```
Comment (AlexeyAkhunov) [inline common/changeset/account_changeset_utils.go:27]: You can take this out of the loop

File: common/dbutils/bucket.go [hunk 2]
```diff
@@ -249,6 +249,9 @@ var DeprecatedBuckets = []string{
 	CurrentStateBucketOld1,
 	PlainStateBucketOld1,
 	IntermediateTrieHashBucketOld1,
+	"uncles",
```
Comment (AlexeyAkhunov) [inline common/dbutils/bucket.go:252]: Is this a left-over from another change?

**Comment by AlexeyAkhunov** [comment]
Looks good to me, I will complete the test of `checkChangeSets` and then a resync of starting from stage5 to see the performance implications

**Comment by AlexeyAkhunov** [comment]
I think CI is going crazy building docker on this branch though. Sending build context of 500 Gb ??

**Comment by mandrigin** [comment]
yeah, and that is after I updated dockerignore :-/

**Comment by AskAlexSharov** [comment]
- Fixed docker problem. 500GB - because some tests are creating database file inside project and I a bit broken in-memory mode.  
- Added to checkChangeSet validation - it now stops if stages Exec or History are lower than block you trying to check
- Continue on checkChangeSet

**Pull request merged**